src/Http/Place/ImportPlaceRequestHandler.php

```diff
@@ -161,7 +161,7 @@ public function handle(ServerRequestInterface $request): ResponseInterface
             $placeAggregate = PlaceAggregate::create(
                 $placeId,
                 $mainLanguage,
-                $title,
+                new Title($title->toString()),
```

Comment:
This seems weird, is this a conversion between the same types?

Reply:
it is not, its a conversion between the old LegacyTitle and the new Title.

I will merge this now, but this will be fixed in the next PR I am going to create in a minute, scope wise I thought it wa s best to do this in a seperate PR.